src/main/java/io/seqera/tower/cli/commands/runs/DumpCmd.java

```diff
@@ -167,15 +170,36 @@ private void dumpTasks(PrintWriter progress, TarArchiveOutputStream out, Long ws
     private void dumpWorkflowDetails(PrintWriter progress, TarArchiveOutputStream out, Long wspId) throws ApiException, IOException {
         progress.println(ansi("- Workflow details"));
 
+        // User info
+        UserDbDto user = api().userInfo().getUser();
```

Comment:
This is the user that is doing the dump. But maybe, at the issue, @ejseqera refers to the user that launched the workflow.

Reply:
Yes, to clarify, we are looking for info of the user that launched the workflow.